build.sh

```diff
@@ -377,11 +384,19 @@ else
             _CC=/usr/bin/clang
             CLANG_PATH=$_CXX
         else
-            echo "ERROR: clang++ not found at /usr/bin/clang++"
-            echo ""
-            echo "You could use clang++ from a custom location."
-            PRINT_USAGE
-            exit 1
+            # try env CXX and CC
+            if [[ ! -f $CXX ]]; then
+                ERROR_CLANG
+            fi
+
+            _CXX=$CXX
+            _CC=$CC
```

Comment:
We weren't validating _CC so far. At least I could validate the availability of the one from ENV. Let me update. Thanks!